src/clr-angular/utils/_close.clarity.scss

```diff
@@ -20,20 +20,20 @@
     font-weight: 200;
     text-shadow: none;
     line-height: inherit;
-    color: $clr-close-color--normal;
+    @include css-var(color, unquote('clr-close-color--normal'), $clr-close-color--normal, $clr-use-custom-properties);
```

Comment:
Is the unquote because sass throws on the double dash `--`?

Reply:
yep